ee/clickhouse/queries/funnels/test/__snapshots__/test_funnel_trends.ambr

```diff
@@ -186,7 +186,7 @@
      GROUP BY entrance_period_start) data
   RIGHT OUTER JOIN
     (SELECT toStartOfDay(toDateTime('2021-04-30 07:00:00') + toIntervalDay(number), 'US/Pacific') AS entrance_period_start
-     FROM numbers(dateDiff('day', toDateTime('2021-04-30 07:00:00'), toDateTime('2021-05-08 06:59:59')) + 1) AS period_offsets) fill USING (entrance_period_start)
+     FROM numbers(dateDiff('day', toDateTime('2021-04-30 07:00:00'), toDateTime('2021-05-07 07:00:00')) + 1) AS period_offsets) fill USING (entrance_period_start)
```

Comment:
hmm, why are these changes happening in my PR? 🤔 

CH version change issue?

Reply:
Yes: https://github.com/PostHog/posthog/pull/9885

Reply:
👍